reference-architecture/vmware-ansible/uninstall_inventory

```diff
@@ -0,0 +1,46 @@
+[OSEv3:children]
+masters
+etcd
+nodes
+ 
+[OSEv3:vars]
+console_port=8443
+openshift_debug_level=2
+openshift_node_debug_level="{{ node_debug_level | default(debug_level, true) }}"
+openshift_master_debug_level="{{ master_debug_level | default(debug_level, true) }}"
+openshift_master_access_token_max_seconds=2419200
+openshift_master_api_port="{{ console_port }}"
+openshift_master_console_port="{{ console_port }}"
+openshift_master_logging_public_url="https://kibana.{{ osm_default_subdomain }}"
+openshift_master_metrics_public_url="https://metrics.{{ osm_default_subdomain }}/hawkular/metrics"
+osm_cluster_network_cidr=172.16.0.0/16
+osm_use_cockpit=false
+openshift_registry_selector="role=infra"
+openshift_router_selector="role=infra"
+openshift_master_cluster_method=native
+openshift_cloudprovider_kind=vmware
+osm_default_subdomain="{{ wildcard_zone }}"
+wildcard_zone="vcenter.e2e.bos.redhat.com"
+deployment_type=openshift-enterprise
+openshift_master_identity_providers="[{'name': 'allow_all', 'login': 'true', 'challenge': 'true', 'kind': 'AllowAllPasswordIdentityProvider'}]"
+ 
+ 
+[masters]
+master-0.vcenter.e2e.bos.redhat.com openshift_node_labels="{'role': 'master'}"
+master-1.vcenter.e2e.bos.redhat.com openshift_node_labels="{'role': 'master'}"
+master-2.vcenter.e2e.bos.redhat.com openshift_node_labels="{'role': 'master'}"
+ 
+[etcd]
+master-0.vcenter.e2e.bos.redhat.com
+master-1.vcenter.e2e.bos.redhat.com
+master-2.vcenter.e2e.bos.redhat.com
+[nodes]
+master-0.vcenter.e2e.bos.redhat.com openshift_node_labels="{'role': 'master'}"
+master-1.vcenter.e2e.bos.redhat.com openshift_node_labels="{'role': 'master'}"
+master-2.vcenter.e2e.bos.redhat.com openshift_node_labels="{'role': 'master'}"
+infra-0.vcenter.e2e.bos.redhat.com openshift_node_labels="{'role': 'infra'}"
+infra-1.vcenter.e2e.bos.redhat.com openshift_node_labels="{'role': 'infra'}"
+primary-0.vcenter.e2e.bos.redhat.com openshift_node_labels="{'role': 'app'}"
+primary-1.vcenter.e2e.bos.redhat.com openshift_node_labels="{'role': 'app'}"
+primary-2.vcenter.e2e.bos.redhat.com openshift_node_labels="{'role': 'app'}"
```

Comment:
This inventory appears to be hardcoded, instead I would use the group mappings for uninstall.